scico/linop/radon_svmbir.py

```diff
@@ -111,7 +139,7 @@ def _bproj_hcb(self, y):
 
 
 class SVMBIRWeightedSquaredL2Loss(WeightedSquaredL2Loss):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, max_iterations=1000, stop_threshold=0.001, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
It looks as if `max_iterations` and `stop_threshold` are just discarded: they're just passed to the `__init__` of `WeightedSquaredL2Loss`, which ignores them.

Reply:
In line 153-154 I use them:
`self.max_iterations = max_iterations`
`self.stop_threshold = stop_threshold`.
These are then used in the recon call.
The `__init__` of `WeightedSquaredL2Loss` just gets the rest of the arguments.

Reply:
But we could alternatively pass these to the prox directly.
What do you think?

Reply:
Oh, for some reason I didn't see that. No, probably best to leave them where they are.